build/NI.CSharp.Analyzers.nuspec

```diff
@@ -27,6 +27,8 @@
   </metadata>
   <files>
     <file src="..\.binaries\Release\NationalInstruments.Analyzers\netstandard2.0\*.dll" target="analyzers\dotnet\cs" />
+    <file src="..\src\AnalyzerConfiguration\*.globalconfig" target="build" />
```

Comment:
I don't think we should put the .globalconfig files in the `build` folder.  I'd probably put it in `contentFiles` - but putting it next to the other files in `content` would also be ok.

Reply:
Unfortunately, `PkgNI_CSharp_Analyzer` isn't defined until *after* the props file is imported, so I'm not able to do `$(PkgNI_CSharp_Analyzer)/content` as the `targets` file was able to do;  I opted to put it in the build folder so that I could use `MSBuildThisFileDirectory`.

However, I guess technically I could do `$(MSBuildThisFileDirectory)../content/NI.CSharp.Analyzer.globalconfig`...I can update that.

Reply:
Yeah - it's not uncommon at all to see that kind of relative pathing in nuget .props/.targets files.

Reply:
Updated to use relative path to content folder.